JustSaying.IntegrationTests/FluentNotificationStackTestBase.cs

```diff
@@ -189,9 +198,17 @@ protected bool QueueHasPolicyForTopic(RegionEndpoint regionEndpoint, Topic topic
         {
             var client = CreateMeABus.DefaultClientFactory().GetSqsClient(regionEndpoint);
 
-            var policy = client.GetQueueAttributes(new GetQueueAttributesRequest{ QueueUrl = queueUrl, AttributeNames = new List<string>{ "Policy" }}).Policy;
+            var policy =
+                client.GetQueueAttributes(new GetQueueAttributesRequest
+                {
+                    QueueUrl = queueUrl,
+                    AttributeNames = new List<string> {"Policy"}
+                }).Policy;
+
+            int pos = topic.TopicArn.LastIndexOf(':');
+            string wildcardedSubscription = topic.TopicArn.Substring(0, pos + 1) + "*";
 
-            return policy.Contains(topic.TopicArn);
+            return policy.Contains(topic.TopicArn) || policy.Contains(wildcardedSubscription);

```

Comment:
presume this test has been broken since the wildcarded sub work was added in previous PRs?